packages/scripts/config/webpack.config.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+const { BundleAnalyzerPlugin } = require( 'webpack-bundle-analyzer' );
+const LiveReloadPlugin = require( 'webpack-livereload-plugin' );
+const path = require( 'path' );
+
+/**
+ * Internal dependencies
+ */
+const { camelCaseDash } = require( '../utils' );
+
+/**
+ * Converts @wordpress/* string request into request object.
+ *
+ * Note this isn't the same as camel case because of the
+ * way that numbers don't trigger the capitalized next letter.
+ *
+ * @example
+ * formatRequest( '@wordpress/api-fetch' );
+ * // { this: [ 'wp', 'apiFetch' ] }
+ * formatRequest( '@wordpress/i18n' );
+ * // { this: [ 'wp', 'i18n' ] }
+ *
+ * @param {string} request Request name from import statement.
+ * @return {Object} Request object formatted for further processing.
+ */
+const formatRequest = ( request ) => {
+	// '@wordpress/api-fetch' -> [ '@wordpress', 'api-fetch' ]
+	const [ , name ] = request.split( '/' );
+
+	// { this: [ 'wp', 'apiFetch' ] }
+	return {
+		this: [ 'wp', camelCaseDash( name ) ],
+	};
+};
+
+const wordpressExternals = ( context, request, callback ) => {
+	if ( /^@wordpress\//.test( request ) ) {
+		callback( null, formatRequest( request ), 'this' );
+	} else {
+		callback();
+	}
+};
+
+const externals = [
+	{
+		react: 'React',
+		'react-dom': 'ReactDOM',
+		tinymce: 'tinymce',
+		moment: 'moment',
+		jquery: 'jQuery',
+		lodash: 'lodash',
+		'lodash-es': 'lodash',
+	},
+	wordpressExternals,
+];
+
+const isProduction = process.env.NODE_ENV === 'production';
+const mode = isProduction ? 'production' : 'development';
+
+const config = {
+	mode,
+	entry: {
+		index: path.resolve( process.cwd(), 'src', 'index.js' ),
+	},
+	output: {
+		filename: '[name].js',
+		path: path.resolve( process.cwd(), 'build' ),
+	},
+	externals,
+	resolve: {
+		alias: {
+			'lodash-es': 'lodash',
+		},
+	},
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				use: require.resolve( 'source-map-loader' ),
+				enforce: 'pre',
+			},
+			{
+				test: /\.js$/,
+				exclude: [
+					/block-serialization-spec-parser/,
```

Comment:
`block-serialization-spec-parser` and `is-shallow-equal` are Gutenberg packages. These exclusions should be added by the Gutenberg's Webpack config.

Reply:
We want to be able to extend these exclusions (perhaps via an env variable?). Some plugins will want to add to these as well, so they can reuse the same mechanism we come up for Gutenberg.

Reply:
Yes, I didn’t have a good idea how to tackle it to be honest. I will think about it tomorrow.

Reply:
It looks like it is no longer necessary. I have no idea why I added it in the first place. Maybe it was a bug in weback which was resolved 🤷‍♂️

Reply:
It wasn't that easy but I fixed it with https://github.com/WordPress/gutenberg/pull/13814/commits/3a8093b31eedb28cbf7b1f35944cbb97012adc2a.